dashboard/app/reporting_email.go

```diff
@@ -179,16 +180,22 @@ func emailSendBugNotif(c context.Context, notif *dashapi.BugNotification) error
 	if err := json.Unmarshal(notif.Config, cfg); err != nil {
 		return fmt.Errorf("failed to unmarshal email config: %v", err)
 	}
-	to := email.MergeEmailLists([]string{cfg.Email}, notif.CC)
+	to := []string{}
+	cc := email.MergeEmailLists([]string{cfg.Email}, notif.CC)
```

Comment:
This is moving `notif.CC` to `Cc` instead of `To` as was previously the case.

It's true it was called "Cc".  But I worry that it was not well thought out, and some emails in it might not actually belong on `Cc`.

To be safe, I'd rather we defaulted to "To" in any legacy cases, and only use "Cc" for entries in the new `Recipients` list, where we explicitly decided that they should go on the email "Cc".

BTW, would it make things any easier to translate the legacy lists into `Recipients` earlier, i.e. immediately when reading from the database or when receiving `dashapi` messages?  I.e. they'd be translated into recipients of type `To`, and the code that actually sends emails would only need to deal with `Recipients`.

Reply:
That makes sense, `notif.CC` contains `Commit.CC` and `kernelRepo.CC` and we want some of the emails. But do we want to do that for all cases? `cfg.Email` contains `syzkaller-bugs` and it should go to Cc.

About translating them, I'm not sure if it would make things easier. We'd be translating them on `reporting.go` and then transforming them to strings again to send the email on `reporting_email.go`. Although there is the benefit of just handling Recipients on `reporting_email.go`. What do you think?

Reply:
Sure, putting `syzkaller-bugs` on the email Cc is fine even for old bugs.  It's the other email addresses I'd be worried about moving to Cc.

My instinct is that translating the Recipients earlier would be easier.  But I'm not sure; it's up to you.